### ISSUE TITLE
Indentation no longer automatic. 

### ISSUE THREAD

**Issue by equinox** [brad-jones/whitespace#1]
I only noticed recently (maybe since changing to 1.3?), that the extension no longer updates all the indentations to Brackets' settings (I like 4, files with 2 are still being 2 when opened).

Possbily related to: [this, which also seems to be having issues](https://github.com/MiguelCastillo/Brackets-wsSanitizer/issues/11)

**Comment by brad-jones** [comment]
I have actually stopped using this. I decided after a little while that sanitizing on file open was perhaps not the best idea. When you start editing files that contain syntax that relies on the indentation being a certain way, for example YAML. This plugin would mangle the file and it quickly became more of a pain in my arse than not.

This plugin needs more smarts to detect the file type and it's current indentation before sanitizing on file open. I just haven't had much time to look at it any further.
